overlord/devicestate/devicemgr.go

```diff
@@ -182,7 +183,8 @@ func (m *DeviceManager) StartUp() error {
 
 	m.state.Lock()
 	defer m.state.Unlock()
-	return EarlyConfig(m.state)
+	// XXX preloadGadget
```

Comment:
Maybe expand this "XXX" a little bit?

Reply:
it's taken care in the follow-up, sorry it's so terse, it should have been:

// XXX implement and provide preloadGadget